src/main/java/com/conveyal/gtfs/error/DuplicateTripError.java

```diff
@@ -12,21 +12,19 @@
     public static final long serialVersionUID = 1L;
 
     public final Priority priority = Priority.LOW;
-    public final String tripId;
     public final String duplicateTripId;
     public final String tripKey;
     public final String routeId;
 
-    public DuplicateTripError(String tripId, String duplicateTripId, String tripKey, String routeId) {
-        super("trips", 0, "trip_id");
-        this.tripId = tripId;
+    public DuplicateTripError(Trip trip, String duplicateTripId, String tripKey) {
+        super("trips", trip.sourceFileLine, "trip_id", trip.trip_id);
         this.duplicateTripId = duplicateTripId;
         this.tripKey = tripKey;
-        this.routeId = routeId;
+        this.routeId = trip.route_id;
 
     }
 
     @Override public String getMessage() {
-        return "Trip Ids " + duplicateTripId + " & " + tripId + " are duplicates (" + tripKey + ")";
+        return String.format("Trip Ids %s & %s (route %s) are duplicates (%s)", duplicateTripId, affectedEntityId, routeId, tripKey);
```

Comment:
I'm just saying for display purposes we should use the human-readable name. Also, we could just check if the trips have the same pattern ID, rather than looping over all the stops.